src/sqlfluff/rules/L010.py

```diff
@@ -9,6 +9,7 @@
     document_fix_compatible,
     document_configuration,
 )
+from sqlfluff.dialects import dialect_ansi as ansi
```

Comment:
Why do we need this import?

```suggestion
```

Reply:
I'm later checking whether the parent is of type ObjectReferenceStatement which is defined in dialect_ansi. Would you prefer a different import?

Reply:
OK but it might not be the ANSI version we want if any dialect overrides it.

Have a look at `_get_table_refs` in L026 which also seems to want the same info as you want, but doesn't do this ANSI import.